### ISSUE TITLE
Errors in configs

### ISSUE THREAD

**Issue by JordanMilly** [ensou04/RocketSoundEnhancementDefault#2]
Currently, on loading into the game, ModuleManager is noting a number of errors:
https://i.imgur.com/CcadnaV.png
4 in GlobalPatches.cfg in the Config folder
4 in Liquid_Heavy-Multi.cfg in the Preset folder
8 in Liquid_Light-Multi.cfg in the Preset folder

I've attached the relevant files, as well as my KSP.log file.
[RSEDefault Error Files.zip](https://github.com/ensou04/RocketSoundEnhancementDefault/files/9590821/RSEDefault.Error.Files.zip)

**Comment by Gordon-Dry** [comment]
I guess it's the same as https://github.com/ensou04/RocketSoundEnhancement/issues/18 which I close over there as it belongs to here.

**Comment by Gordon-Dry** [comment]
I needed several runs and tests to all the same config:
GameData\RocketSoundEnhancementDefault\Presets\000_PresetDefaults.cfg
to tailor out all remaining MM errors.

But it's always the same, only one occasion of a fitting part config is parsed without errors, the others are faulty.
In my case, this one is good:
`GameData\KerbalAtomics\Parts\NuclearEngines\ntr-375\ntr-sc-375-1.cfg`

and those 3 are bad:
```
GameData\KerbalAtomics\Parts\NuclearEngines\ntr-375\ntr-sc-25-1.cfg
GameData\KerbalAtomics\Parts\NuclearEngines\ntr-375\ntr-sc-125-1.cfg
GameData\KerbalAtomics\Parts\NuclearEngines\ntr-375\ntr-sc-125-2.cfg
```

I see the same syntax in all of them and the affected patches so I don't get it what should be the issue.

My last attempt of trying to fix it was this
```
GameData\RocketSoundEnhancementDefault\Presets\000_PresetDefaults.cfg
@PART[*]:HAS[@RSE_PRESET[*],@MODULE[ModuleEngines*]:HAS[~engineID]]:FOR[zRocketSoundEnhancementDefault]:NEEDS[RocketSoundEnhancement]
{
	@RSE_PRESET,*
	{
        %engineID = Engine
		%volume = 1.00
		%pitch = 1.00
	}
}

@PART[*]:HAS[@RSE_PRESET[*],@MODULE[ModuleEngines*]:HAS[#engineID]]:FOR[zRocketSoundEnhancementDefault]:NEEDS[RocketSoundEnhancement]
{
	@RSE_PRESET,*
	{
        %engineID = #$/MODULE[ModuleEngines*]:HAS[#engineID]/engineID$
		%volume = 1.00
		%pitch = 1.00
	}
}

@PART[*]:HAS[@RSE_PRESET[*Multi],@MODULE[MultiModeEngine]]:FOR[zRocketSoundEnhancementDefault]:NEEDS[RocketSoundEnhancement]
{
	@RSE_PRESET[*Multi]
	{
		%primaryEngineID = PrimaryEngine
		@primaryEngineID = #$/MODULE[MultiModeEngine]:HAS[#primaryEngineID]/primaryEngineID$
		%secondaryEngineID = SecondaryEngine
		@secondaryEngineID = #$/MODULE[MultiModeEngine]:HAS[#secondaryEngineID]/secondaryEngineID$
		%primaryEngine_VolumeScale = 1.0
		%secondaryEngine_VolumeScale = 1.0
	}
}
```
but to no avail.

**Comment by TheOneTrueMongoloid** [comment]
I had this exact same issue with Kerbal Atomics mod as well.

**Comment by Clayell** [comment]
I am also having this issue with kerbal atomics, its always 12 errors that are shown:
"4 errors related to GameData/RocketSoundEnhancementDefault/Presets/Liquid_Heavy-Multi.cfg"
"8 errors related to GameData/RocketSoundEnhancementDefault/Presets/Liquid_Light-Multi.cfg"

**Comment by JonnyOThan** [comment]
@TheOneTrueMongoloid  and @Clayell Hi, please include your ksp.log file.  If you're not sure where it is, see here: https://gist.github.com/JonnyOThan/04c2074b56f78e56d115621effee30f9

I can't repro this by installing just RSE and KerbalAtomics, so if someone could provide a minimal set of mods to install I'd appreciate it.

**Comment by Clayell** [comment]
Sure, here it is: https://drive.google.com/file/d/10FeUGZCGs04WPSUCKFbGlaJBAuCmtsii/view?usp=sharing (KSP.log)

I also made a note of the list of mods that I installed in a batch and then first encountered this bug with.
> kerbal atomics, heat control, near future launch vehicles, near future propulsion, ro engines extended

**Comment by JonnyOThan** [comment]
OK got a repro with these mods:

```
Advanced Jet Engine (AdvancedJetEngine v2.20.0)
B9 Part Switch (B9PartSwitch v2.20.0)
ClickThrough Blocker (ClickThroughBlocker 1:2.1.10.21)
Community Resource Pack (CommunityResourcePack v112.0.1)
Cryo Tanks (CryoTanks 1.6.5)
Cryo Tanks Core (CryoTanks-Core 1.6.5)
Deployable Engines Plugin (DeployableEngines 1.3.1)
Dynamic Battery Storage (DynamicBatteryStorage 2:2.2.5.0)
Ferram Aerospace Research Continued (FerramAerospaceResearchContinued 3:0.16.1.2)
Harmony 2 (Harmony2 2.2.1.0)
Heat Control (HeatControl 0.6.1)
Kerbal Atomics (KerbalAtomics 1:1.3.3)
Kerbal Changelog (KerbalChangelog v1.4.2)
Kerbal Joint Reinforcement Continued (KerbalJointReinforcementContinued v3.7.1.0)
KSP Community Fixes (KSPCommunityFixes 1.30.0)
KSPBurst Lite (KSPBurst-Lite v1.5.5.1)
ModularFlightIntegrator (ModularFlightIntegrator 1.2.10.0)
Module Manager (ModuleManager 4.2.3)
Near Future Launch Vehicles (NearFutureLaunchVehicles 2.2.0)
Near Future Propulsion (NearFuturePropulsion 1.3.5)
Patch Manager (PatchManager 0.0.17.6)
QuickStart (QuickStart 1:2.2.1.1)
Real Fuels (RealFuels 1:v15.6.3.0)
Real Plume (RealPlume 2:v13.3.2)
RealChute Parachute Systems (RealChute v1.4.8.3)
RealHeat (RealHeat v5.1)
Realism Overhaul (RealismOverhaul v17.4.0.0)
RO Engines (ROEngines v2.5.0.0)
RO Library (ROLib v1.7.3.0)
ROEngines Extended (ROEnginesExtended 2.1.0)
SmokeScreen - Extended FX Plugin (SmokeScreen 2.8.14.0)
Solver Engines plugin (SolverEngines v3.14.0)
Textures Unlimited (TexturesUnlimited 1.5.10.25)
Toolbar Controller (ToolbarController 1:0.1.9.11)
TweakScale Redistributable (TweakScale-Redist v2.4.7.3)
Waterfall Core (Waterfall 0.9.0)
```
